src/qrcode/QRTabBar.tsx

```diff
@@ -41,7 +41,7 @@ export default function QRTabBar({
     [state, descriptors]
   )
 
-  const color = state.index === 0 ? colors.white : colors.darkBlue
+  const color = state.index === 0 ? colors.white : colors.white
```

Comment:
this and a few others can be cleaned up better, but keeping the diff related to colors minimal for now